pvlib/spectrum/mismatch.py

```diff
@@ -235,3 +236,159 @@ def integrate(e):
         smm = pd.Series(smm, index=e_sun.index)
 
     return smm
+
+
+def martin_ruiz_spectral_modifier(clearness_index, airmass_absolute,
```

Comment:
I would prefer `spectrum.martin_ruiz`, even though that creates potential for namespace conflict with `pvlib.iam.martin_ruiz` if a user imports without the module name.

Reply:
Shall I wait for more input?